base/util.jl

```diff
@@ -638,7 +638,7 @@ will always be called.
 function securezero! end
 @noinline securezero!(a::AbstractArray{<:Number}) = fill!(a, 0)
 securezero!(s::String) = unsafe_securezero!(pointer(s), sizeof(s))
-@noinline unsafe_securezero!{T}(p::Ptr{T}, len::Integer=1) =
+@noinline unsafe_securezero!(p::Ptr{T}, len::Integer=1) where T =
```

Comment:
It's now pretty easy to do a cleanup pass looking for `r"\)\s*where\s+\w+\s*="` anywhere.